src/discord/discord.py

```diff
@@ -46,6 +46,13 @@ async def _discord_request(
             'User-Agent' : 'DiscordBot (https://github.com/CSSS/csss-site-backend, 1.0)'
         }
     )
+    rate_limit_reset = float(result.headers["x-ratelimit-reset-after"])
+    rate_limit_remaining = int(result.headers["x-ratelimit-remaining"])
```

Comment:
is this number of remaining requests? if so, can you rename to `rate_limit_remaining_requests`, i see now why the number is 2. If there were 10 threads and we accidentally got rate limited, would this entry be 0, or would it not exist?

Reply:
The headers change completely iirc

Reply:
Resolved in f777c12

Reply:
> The headers change completely iirc

if the headers change completely, then this function would raise an exception for failing to index the x-ratelimit-remaining entry? Maybe we should handle that exception with a custom error?

Reply:
Ah shoot, I was looking at the wrong PR.  The headers do not change.